### PR TITLE
Baseline with the most popular movies

### PR DESCRIPTION
**Pull request by atsanda**
I added a model which recommends the most popular movies from the dataset

### CODE REVIEW

File: moviesurfer/models/popular.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        The method excludes already viewed movies mentioned in the references.
+
+        Args:
+            references (list[int]): A list of already chosen item IDs.
+            n (int): The number of recommendations to generate.
+
+        Returns:
+            list[int]: A list of item IDs representing the recommendations.
+        """
+        references = set(references or [])
```
Comment (bq777xCube) [inline moviesurfer/models/popular.py]: We could change references into something like "excluded movies" to make it more clear what it represents, but it's up to you.

File: moviesurfer/models/popular.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        recs = []
+        for i, movie in enumerate(cycle(self.popularity)):
```
Comment (bq777xCube) [inline moviesurfer/models/popular.py:72]: Also, you could add a brief comment explaining what the cycle function does as it's not a common syntax.

**Review by bq777xCube — APPROVED** [review]

File: moviesurfer/models/popular.py (hunk 1 above)
Comment (atsanda) [reply moviesurfer/models/popular.py]: agree
changed
please, resolve if you agree with the changes

File: moviesurfer/models/popular.py (hunk 2 above)
Comment (atsanda) [reply moviesurfer/models/popular.py:72]: agree
changed
please, resolve if you agree with the changes

File: moviesurfer/models/popular.py (hunk 1 above)
Comment (bq777xCube) [reply moviesurfer/models/popular.py]: Thank you. Please also change it the docstring explanation and I think you can merge it.

**Review by bq777xCube — APPROVED** [review]

**Pull request merged**